shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/user/Profile.kt

```diff
@@ -0,0 +1,10 @@
+package org.itmodreamteam.myrest.shared.user
+
+data class Profile(
+    val id: Long,
+    val firstName: String,
+    val lastName: String,
+    val enabled: Boolean,
+    val locked: Boolean,
+    val role: Role?,
```

Comment:
How are we gonna set the role?

Reply:
Setting the role of system administrator can be done manually via database :)
OR we can automatically create this user at start up if it doesn't exist.